config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
-  - manager.yaml
```

Comment:
there should not be changes in this file and also changes to the image in ./bundle/manifests/